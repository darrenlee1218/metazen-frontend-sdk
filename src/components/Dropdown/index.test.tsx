import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { TestProviders } from '@utils/testing/TestProviders';
import { Dropdown } from '.';

afterEach(cleanup);

const onChange = jest.fn();
it('matches snapshot', () => {
  const { container } = render(<Dropdown list={[]} onChange={onChange} />, { wrapper: TestProviders });

  expect(container).toMatchSnapshot();
});
