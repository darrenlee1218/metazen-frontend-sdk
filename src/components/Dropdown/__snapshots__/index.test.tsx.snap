// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="MuiFormControl-root css-aekpvq-MuiFormControl-root"
  >
    <div
      class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall  css-wd7n1v-MuiInputBase-root-MuiOutlinedInput-root-MuiSelect-root"
    >
      <div
        aria-expanded="false"
        aria-haspopup="listbox"
        class="MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1t2t4xt-MuiSelect-select-MuiInputBase-input-MuiOutlinedInput-input"
        role="button"
        tabindex="0"
      >
        <span
          class="notranslate"
        >
          ​
        </span>
      </div>
      <input
        aria-hidden="true"
        class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
        tabindex="-1"
        value=""
      />
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiSelect-icon MuiSelect-iconOutlined css-1wu7ecg-MuiSvgIcon-root-MuiSelect-icon"
        data-testid="KeyboardArrowDownIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M7.41 8.59 12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
        />
      </svg>
      <fieldset
        aria-hidden="true"
        class="MuiOutlinedInput-notchedOutline css-700zfh-MuiOutlinedInput-notchedOutline"
      >
        <legend
          class="css-hdw1oc"
        >
          <span
            class="notranslate"
          >
            ​
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
</div>
`;
